ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/questionMark.html

```diff
@@ -1,4 +1,6 @@
-<template data-sly-template.questionMark="${@ componentId, longDescription, bemBlock}">
-  <button class="${bemBlock}__questionmark" data-sly-test="${longDescription}" title="Help text">
+<template data-sly-template.questionMark="${@ componentId, longDescription, bemBlock, componentTitle}">
+  <sly data-sly-set.namedTitle="${componentTitle} help text" />
+  <sly data-sly-set.fullTitle="${componentTitle ? namedTitle : 'Help text'}" />
```

Comment:
Since we are fixing this can we add localization fix as well, use following instead:
 ${'Help text' @ i18n}